files/zh-cn/web/javascript/reference/global_objects/array/reduceright/index.md

```diff
@@ -20,159 +18,130 @@ reduceRight(callbackFn, initialValue)
 
 ### 参数
 
-- `callback`
-  - : 一个回调函数，用于操作数组中的每个元素，它可接受四个参数：
+- `callbackFn`
+  - : 为数组中的每个元素执行的函数。其返回值将作为下一次调用 `callbackFn` 时的 `accumulator` 参数。对于最后一次调用，返回值将成为 `reduceRight()` 的返回值。该函数被调用时将传入以下参数：
     - `accumulator`
-      - : 累加器：上一次调用回调函数时，回调函数返回的值。首次调用回调函数时，如果 `initialValue` 存在，累加器即为 `initialValue`，否则须为数组中的最后一个元素（详见下方 `initialValue` 处相关说明）。
+      - : 上一次调用回调函数时返回的值，如果指定了 `initialValue` 则为指定的值。（见下文）
```

Comment:
这么翻译是不正确的，`initialValue` 仅在第一次调用时（若提供）才会作为累加器的值，但现在的翻译变成了“每一次”调用均为这个值（若提供）

Reply:
原文中是“or `initialValue`, if supplied. (See below.)”，这种歧义在原文中也存在。我会尝试修改原文。

Reply:
原文只是没有说明，仅在第一次调用时才会使用该值，但说明并没有问题，它用的是“或”

Reply:
如果仅是按照原文的话，我的理解这里的“or”表转折说明另一种情况，即：（对于第一次调用时，）如果指定了 `initiaValue` 的值，则 `accumulator` 为指定的值（，否则为 `array[this.length - 1]`）。我认为原文中这里也并没有详细说明这种情况，而是通过“See below”来指明将在下文具体说明。
而在“reduce”中，对应 `accumulator` 的参数说明原文是“The value resulting from the previous call to `callbackFn`. On first call, `initialValue` if specified, otherwise the value of `array[0]`”，这句原文则说明了第一次调用时的特殊情况，我认为歧义性在原文中也存在了，所以我的 [PR](https://github.com/mdn/content/pull/26594) 将“reduceRight”中 `acumulator` 的描述修改为与“reduce”中保持一致性的描述，毕竟“reduceRight”原文很多内容也是借鉴“reduce”的:D